docs/source/examples/workspace/topologies/aws-ec2-new.yml

```diff
@@ -12,3 +12,5 @@ resource_groups:
         security_group: "default"
         count: 1
         keypair: "herlo_key"
+        assign_public_ip: false
+        vpc_subnet_id: "subnet-29e1aa2c"
```

Comment:
We are working on getting tests in place for each of the providers. These example (things that end in -new) topologies are what we use with the tests. Will this subnet settings work for a test with generic credentials? I would prefer not to mess with the examples here, but create a new topology that shows these changes. Would you mind making that change?

If desired, the topology you create could also be linked as an example from the docs. This would bolster the examples for aws.

Reply:
Ah, I didn't realize that these topologies were used for tests.  Those subnet settings will likely not work unless you've set up a VPC with proper network configuration for your AWS account, which is no small task.  

I can create a new file in the same directory without the '-new' part of the filename and link it to the docs.  Alternatively, we could just comment those lines out of the existing file with a #noqa or something like that.  Thoughts?

Reply:
Yeah, it's not completely clear we're using them unless you look at the testing itself. I figure I'll be creating '-test' versions soon anyway. But we eventually want all of the examples to be validated regularly so we know that they are good when a user copies them as a model.

Creating a different example topology file will be better methinks. We can eventually set it up to do proper testing, and it doesn't add unnecessary lines for a new user.

Reply:
Thanks for the info.  I've made the recommended changes.